### PR TITLE
Updates to default welcome page.

### PR DESCRIPTION
**Pull request by jxtx**
Reference CONTRIBUTORS.md, update institution references, update to
HTML5, use some standard classes for formatting, include links for
configuring and installling tools.

### CODE REVIEW

**Comment by martenson** [comment]
Nice changes.
I am not a fan of underlined text in buttons and their links need to have target added so they won't open in the middle pane.

preview for others to quickcheck:
screenshot 2015-11-01 10 06 34

**Comment by jxtx** [comment]
> I am not a fan of underlined text in buttons

I agree, I was just using the default styles. Could style this as action-button, but it seems to make sense that an a.btn should never be underlined so I went with that.

> and their links need to have target added so they won't open in the middle pane.

Done.

File: static/welcome.html.sample [hunk 1]
```diff
@@ -1,28 +1,43 @@
...
+            with the support of  <a target="_blank" class="reference" href="https://github.com/galaxyproject/galaxy/blob/dev/CONTRIBUTORS.md">many contributors</a>.
+        </p>
+
+        <footer>
+            The <a target="_blank" class="reference" href="http://galaxyproject.org/">Galaxy Project</a>
+            is supported in part by <a target="_blank" class="reference" href="http://www.genome.gov">NHGRI</a>,
+            <a target="_blank" class="reference" href="http://www.nsf.gov">NSF</a>,
+            <a target="_blank" class="reference" href="http://www.huck.psu.edu">The Huck Institutes of the Life Sciences</a>,
+            <a target="_blank" class="reference" href="http://www.ics.psu.edu">The Institute for CyberScience at Penn State</a>,
+            and <a target="_blank" class="reference" href="http://www.emory.edu/">Johns Hopkins University</a>.
```
Comment (martenson) [inline static/welcome.html.sample]: jhu link leads to emory

**Comment by hexylena** [comment]
@jxtx / @martenson with the new welcome.html which I've just noticed, should step 1 of "installing tools" really be "configure a toolshed" (since a very sane default is already provided)? 
https://wiki.galaxyproject.org/Admin/Tools/AddToolFromToolShedTutorial

**Comment by martenson** [comment]
@erasche I changed the wiki, does that seem alright now?

**Comment by hexylena** [comment]
@martenson awesome, thanks! :)

**Pull request merged**